cookbook-react/src/recipes/selectors/selector-2/styles.scss

```diff
@@ -0,0 +1,38 @@
+$color-gray: #E1E1E1;
```

Comment:
as this component is a react component you should name this file `styles.module.scss`